clients/src/main/resources/common/message/OffsetFetchRequest.json

```diff
@@ -39,6 +41,8 @@
         "about": "The topic name."},
       { "name": "PartitionIndexes", "type": "[]int32", "versions": "0+",
         "about": "The partition indexes we would like to fetch offsets for." }
-    ]}
+    ]},
+    {"name": "WaitTransaction", "type": "bool", "versions": "7+", "default": "false",
```

Comment:
Having "wait" in the name sort of suggests to me that the request itself will await pending transactions. I wonder if instead this should be something like `requireResolved` or `requireStable`.

Reply:
I see your point, but I will take this as a nit, because we actually used `waitTransaction` in the KIP already.

Reply:
Minor naming changes to the KIP, even after approval, are reasonable I think.

Reply:
Cool